tools/hatch-cli/src/templates/client-sdk/template/README.md

```diff
@@ -0,0 +1,33 @@
+# HATCH_CLI_TEMPLATE_VAR_projectShortName
+This project is an autogenerated TypeScript client sdk project based on an [OpenAPI Specification](https://swagger.io/specification/).
```

Comment:
I'd reword to indicate that only some of the files are autogenerated. For many libraries, there might be a wrapper class that is handwritten.

Reply:
updated